Snippets/Metrics/Metrics_1/Configuration.cs

```diff
@@ -44,6 +44,28 @@ public void ConfigureEndpoint(EndpointConfiguration endpointConfiguration)
                 interval: TimeSpan.FromSeconds(5));
 
             #endregion
+
+            #region Metrics-Observers [1.1,)
```

Comment:
partials are derived using the dirs. so to have a `metrics_reporting_metrics_[1.1,).partial.md` you would need a 1.1 here https://github.com/Particular/docs.particular.net/tree/metrics-1.1-bump-up/Snippets/Metrics